### PR TITLE
add moist hs to long run

### PR DESCRIPTION
**Pull request by jiahe23**
add moist hs to long run
- central differential set up; dt = 200

### CODE REVIEW

File: .buildkite/longruns/pipeline.yml [hunk 1]
```diff
@@ -43,6 +43,12 @@ steps:
         artifact_paths: "longrun_hs_rhoe/*"
         agents:
           slurm_cpus_per_task: 8
+          
+      - label: ":computer: held suarez (ρe) equilmoist"
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --moist equil --vert_diff true --microphy 0M --dt 200secs --t_end 1200days --upwinding none --fps 30 --job_id longrun_hs_rhoe --dt_save_to_sol 6hours"
+        artifact_paths: "longrun_hs_rhoe_equil/*"
```
Comment (charleskawczynski) [inline .buildkite/longruns/pipeline.yml:49]:
```suggestion
        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --moist equil --vert_diff true --microphy 0M --dt 200secs --t_end 1200days --upwinding none --fps 30 --job_id longrun_hs_rhoe_equil --dt_save_to_sol 6hours"
        artifact_paths: "longrun_hs_rhoe_equil/*"
```

**Review by charleskawczynski — APPROVED** [review]
LGTM with the suggested change (the `job_id`s need to be unique)

File: .buildkite/longruns/pipeline.yml [hunk 2]
```diff
@@ -43,6 +43,12 @@ steps:
         artifact_paths: "longrun_hs_rhoe/*"
         agents:
           slurm_cpus_per_task: 8
+          
+      - label: ":computer: held suarez (ρe) equilmoist"
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --moist equil --vert_diff true --microphy 0M --dt 200secs --t_end 1200days --upwinding none --fps 30 --job_id longrun_hs_rhoe_equil --dt_save_to_sol 6hours"
```
Comment (jiahe23) [inline .buildkite/longruns/pipeline.yml:48]:
```suggestion
        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --moist equil --vert_diff true --microphy 0M --dt 200secs --t_end 600days --upwinding none --fps 30 --job_id longrun_hs_rhoe_equil --dt_save_to_sol 6hours"
```

**Pull request merged**